plaza/routes/exec/exec.go

```diff
@@ -0,0 +1,61 @@
+package exec
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/Nanocloud/community/plaza/windows"
+	"github.com/labstack/echo"
+)
+
+type bodyRequest struct {
+	Username string   `json:"username"`
+	Password string   `json:"password"`
+	Domain   string   `json:"domain"`
+	Command  []string `json:"command"`
+	Stdin    string   `json:"stdin"`
+}
+
+func Route(c *echo.Context) error {
+	b, err := ioutil.ReadAll(c.Request().Body)
+	if err != nil {
+		return err
+	}
+
+	body := bodyRequest{}
+	err = json.Unmarshal(b, &body)
+	if err != nil {
+		return err
+	}
+
+	cmd := windows.Command(
+		body.Username, body.Domain, body.Password,
+		body.Command[0], body.Command[1:]...,
+	)
+
+	if body.Stdin != "" {
+		cmd.Stdin = strings.NewReader(body.Stdin)
+	}
+
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err = cmd.Run()
+
+	// if err != nil {
```

Comment:
When the process fails (Exit code != 0), it returns an error but usefull informations are in stdout (for powershell for instance). But the error can be something else that an invalid exit code. We should deal with that case in the future.